app/models/foreman_docker/docker.rb

```diff
@@ -29,12 +34,11 @@ def max_cpu_count
     end
 
     def available_images
-      client.images.all
+      ::Docker::Image.all({}, docker_connection)
     end
 
     def local_images(filter = '')
-      client # initialize Docker-Api
-      ::Docker::Image.all('filter' => filter)
+      ::Docker::Image.all({ 'filter' => filter }, docker_connection)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.